pay-id-api-spec/pay-id.v1.yml

```diff
@@ -148,21 +148,38 @@ components:
           type: string
       required:
         - address
-    PaymentInformation:
-      title: PaymentInformation
+    Address:
```

Comment:
Can we make this "PayIdAddress" in the SDK swagger? Then you might not need to rename `XRPAddress`, right?

Reply:
yeah this is a good point... comments below, and want to ask Keefer before changing this.

Reply:
This isn't really an 'Address' It's more of "AddressAndMetadata"? I actually don't have a great name here: "AddressResult"?

Reply:
Yeah, the actual "address" here is the `addressDetails`, right?

I'm ok with any name you guys think is appropriate. My vote would still be for `PayIdAddress`, which can encapsulate the concept that it is both metadata around a payment rail address, and the payment rail address itself.